masterlist.yaml

```diff
@@ -3190,16 +3190,21 @@ plugins:
         condition: 'active("NAT.esp") and not active("Audio Overhaul Skyrim - Natural Atmospheric Tamriel Patch.esp") and not active("NAT_TS_Patch_.*\.esp")'
       - <<: *patchProvided
         subs: [ 'Obsidian Weathers and Seasons' ]
-        condition: 'active("Obsidian Weathers.esp") and not active("Audio Overhaul Skyrim - Obsidian Weathers Patch.esp") and not active("Obsidian_TS_Patch__.*\.esp")'
+        condition: 'active("Obsidian Weathers.esp") and not active("Audio Overhaul Skyrim - Obsidian Weathers Patch.esp") and not active("TrueStormsSE.esp") and not active("Obsidian_TS_Patch__.*\.esp")'
       - <<: *patchProvided
         subs: [ 'Rustic Weathers and Lighting' ]
         condition: 'active("Rustic Weathers And Lighting.esp") and not active("Audio Overhaul Skyrim - Rustic Weathers Patch.esp") and not active("Rustic_TS_Patch_.*\.esp")'
       - <<: *patchProvided
         subs: [ 'True Storms - Thunder, Rain & Weather Redone' ]
-        condition: 'active("TrueStormsSE.esp") and not active("Audio Overhaul Skyrim - True Storms Patch.esp") and not active("(Aequinoctium|ELFXWeathers|Mythical|NAT|Obsidian|Rustic|Vivid)_TS_Patch_.*\.esp")'
+        condition: 'active("TrueStormsSE.esp") and not active("Audio Overhaul Skyrim - True Storms Patch.esp") and not active("Obsidian Weathers.esp") and not active("(Aequinoctium|ELFXWeathers|Mythical|NAT|Obsidian|Rustic|Vivid)_TS_Patch_.*\.esp")'
```

Comment:
If I recall correctly you can use Obsidian, AOS and True Storm together with the official patches.
Though you may still need to do some of your own patches for it to work 100%.
So I don't see why we should be hiding messages if the official patches work together.